python_utils_test.py

```diff
@@ -150,6 +154,72 @@ def test_url_join(self):
             'http://www.cwi.nl/%7Eguido/Python.html', 'FAQ.html')
         self.assertEqual(response, 'http://www.cwi.nl/%7Eguido/FAQ.html')
 
+    def test_recursively_convert_to_str_with_dict(self):
+        test_var_1 = python_utils.UNICODE('test_var_1')
+        test_var_2 = python_utils.UNICODE('test_var_2')
```

Comment:
Split each of these numbered variables into a well-named test case.

Reply:
Done